larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -520,7 +543,8 @@ INSTANTIATE_TEST_SUITE_P(
     BConv2DTests, BConv2DOpTest,
     // WARNING: ::testing::Combine accepts max 10 arguments!!!
     ::testing::Combine(
-        ::testing::Values(1),  // batches
+        ::testing::Values(ActivationFunctionType_NONE,
```

Comment:
here you are removing the test of of `batches` in order to add the test for activation!!! I do think we should keep the test for it and add a new test which coverers a different combination of testing arguments. maybe you can use test fixture to avoid code duplication.

Reply:
:+1: 

My reasoning was this: the `Combine` thing only accepts 10 arguments and `batches` was always 1, its not like we were "testing for batches". If we copy this stuff to a new test, we would have more code duplication (all the test parameters for example), so that's why I did it like this.

I think you argument is valid, but if we change it, then we should also test for different batch sizes.

Instead of creating multiple tests, what do you think about fusing some other parameters into a single one. For example filter count could be merged with filter height/width, so then instead of
```cpp
::testing::Values(std::array<int, 2>{1, 1},
                  std::array<int, 2>{3, 3},
                  std::array<int, 2>{2, 3}),  // filter height/width
::testing::Values(1, 4, 64),                  // filter count
```
we would have something like this for example:
```cpp
::testing::Values(std::array<int, 3>{1, 1, 1},
                  std::array<int, 2>{4, 3, 3},
                  std::array<int, 3>{64, 3, 3},
                  std::array<int, 3>{4, 2, 3},
                  std::array<int, 3>{64, 2, 3}
),  // filter count,height,width
```

This way we can still use a single test. I'm also fine with creating two separate tests and having some duplication with test parameters, just wanted to get your opinion.

Reply:
agreed but in this way only hard-coded combinations will be tested and not all possible combinations! can you check if there is a way to generate these combinations?